test_lib/compaction.py

```diff
@@ -110,3 +119,34 @@ def get_compaction_random_additional_params(strategy: CompactionStrategy):
              {'sstable_size_in_mb': sstable_size_in_mb}]
     }
     return list_additional_params_options[strategy]
+
+
+def calculate_allowed_twcs_ttl_borders(compaction_info: dict, default_min_ttl: int):
+    # Get compaction_window_size and unit
```

Comment:
redundant comment.
Please place comment why we need that and what are the rules for allowed_twc_ttl_borders as I couldn't find it